docs/i18n/ja/docusaurus-plugin-content-docs/current/architecture-overview.md

```
@@ -0,0 +1,228 @@
+    Library->>Provider: Access Storage
+
+    Provider->>KMS: Sign Credential
+
+    KMS-->>Provider: Signature
+
+    Provider-->>Library: Credential
+
+    Library-->>Wallet: Verifiable Credential
+```
+
+### 処理の流れ
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)